### DISCUSSION TITLE
favoris photos as a folder, not only an album

### DISCUSSION THREAD

**Discussion by faon-92** [deckerst/aves#816] [Ideas]
Hi,
I'm wondering if it's possible de make the favoris photos into a folder, not only an album, so I can sync this folder to my drive.
I find no galery app in the play store, being able to do so...

thanks for your feedback,

**Comment by JanWaldhorn** [comment]
Hi @faon-92 
you may have a look at #407 and #143.
I personally use the shortcut to move an image to a favorites folder (which I then synchronize). I don't even use the favorites function anymore and it's practically as quick/easy as assigning a heart.

**Comment by faon-92** [comment]
thank you for this quick response, I will dive into to find a suitable
solution for myself.
Have a nice day

On Mon, Nov 27, 2023 at 1:22 PM JanWaldhorn ***@***.***>
wrote:

> Hi @faon-92 <https://github.com/faon-92>
> you may have a look at #407 <https://github.com/deckerst/aves/issues/407>
> and #143 <https://github.com/deckerst/aves/issues/143>.
> I personally use the shortcut to move an image to a favorites folder
> (which I then synchronize). I don't even use the favorites function anymore
> and it's practically as quick/easy as assigning a heart.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/deckerst/aves/discussions/816#discussioncomment-7680301>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6MWTDAUOTJO365RTU66JTTYGSAZHAVCNFSM6AAAAAA734GPLOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMOBQGMYDC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Feng Sridi
